src/Sulu/Bundle/CustomUrlBundle/EventListener/RequestAnalyzerLocalizationEnhancer.php

```diff
@@ -1,37 +0,0 @@
-<?php
-
-/*
- * This file is part of Sulu.
- *
- * (c) MASSIVE ART WebServices GmbH
- *
- * This source file is subject to the MIT license that is bundled
- * with this source code in the file LICENSE.
- */
-
-namespace Sulu\Bundle\CustomUrlBundle\EventListener;
-
-use Sulu\Component\Webspace\Analyzer\EnhancerInterface;
-use Sulu\Component\Webspace\Analyzer\RequestAnalyzerInterface;
-use Symfony\Component\HttpFoundation\Request;
-
-/**
- * Update localization in case of custom-url route.
- */
-class RequestAnalyzerLocalizationEnhancer implements EnhancerInterface
```

Comment:
Are you sure this class can be deleted? :thinking:

Reply:
The `EnhancerInterface` does not exist and the method called `$requestAnalyzer->updateLocale` also does not exist. also no service is registered.